### PR TITLE
{chem}[GCC/10.2.0,GCC/10.3.0,GCC/11.2.0,GCC/11.3.0,GCC/12.2.0,GCC/9.3.0] NBO v7.0.10

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    'NBO-%(version)s_make.patch',
+]
+checksums = [
+    {'nbo7.0.10-src.tar.gz': 'a60d1f4d4592ec386b5378841d9fba75918c48b50de08d8de0a50db42bc96da1'},
+    {'NBO-7.0.10_make.patch': 'c37f5a9980c56198f9610611b0fbda8ad65c961ec19b163d602dc3330fced145'},
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.0.4'),
+    ('OpenBLAS', '0.3.17'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.3.0.eb:28]:
```suggestion
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    'NBO-%(version)s_make.patch',
+]
+checksums = [
+    {'nbo7.0.10-src.tar.gz': 'a60d1f4d4592ec386b5378841d9fba75918c48b50de08d8de0a50db42bc96da1'},
+    {'NBO-7.0.10_make.patch': 'c37f5a9980c56198f9610611b0fbda8ad65c961ec19b163d602dc3330fced145'},
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.0.4'),
+    ('OpenBLAS', '0.3.18'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.2.0.eb:28]:
```suggestion
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    'NBO-%(version)s_make.patch',
+]
+checksums = [
+    {'nbo7.0.10-src.tar.gz': 'a60d1f4d4592ec386b5378841d9fba75918c48b50de08d8de0a50db42bc96da1'},
+    {'NBO-7.0.10_make.patch': 'c37f5a9980c56198f9610611b0fbda8ad65c961ec19b163d602dc3330fced145'},
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.2.0'),
+    ('OpenBLAS', '0.3.20'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.3.0.eb:28]:
```suggestion
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-12.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    'NBO-%(version)s_make.patch',
+]
+checksums = [
+    {'nbo7.0.10-src.tar.gz': 'a60d1f4d4592ec386b5378841d9fba75918c48b50de08d8de0a50db42bc96da1'},
+    {'NBO-7.0.10_make.patch': 'c37f5a9980c56198f9610611b0fbda8ad65c961ec19b163d602dc3330fced145'},
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.2.1'),
+    ('OpenBLAS', '0.3.21'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-12.2.0.eb:28]:
```suggestion
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
Using the `GCC` toolchain and adding `FlexiBLAS` or `OpenBLAS` as dependency is possible, but I would rather use toolchain that already contains a BLAS library.

Please note that adding `FlexiBLAS` and `OpenBLAS` at the same time is not possible, only `FlexiBLAS` should be used.

From 2022a onward we added the toolchain `gfbf` (GCC + FlexiBLAS + FFTW serial). This might be a good choice for a software that needs `GCC` and a `BLAS` library.

**Comment by sassy-crick** [comment]
Thanks for looking into that. I done the suggested changes and done some polishing of the files as well. The patchfile sets `OpenBLAS` as the default BLAS, so I removed that where appropriated in the EC file. I hope it makes sense.

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.2.0.eb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
+
+# NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+# The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.2.0.eb:16]:
```suggestion
download_instructions = """
NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
"""
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.3.0.eb [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
+
+# NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+# The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-10.3.0.eb:16]:
```suggestion
download_instructions = """
NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
"""
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.2.0.eb [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
+
+# NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+# The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-11.2.0.eb:16]:
```suggestion
download_instructions = """
NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
"""
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-9.3.0.eb [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
+
+# NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+# The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-GCC-9.3.0.eb:16]:
```suggestion
download_instructions = """
NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
"""
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-gfbf-2022a.eb [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'gfbf', 'version': '2022a'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
+
+# NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+# The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-gfbf-2022a.eb:16]:
```suggestion
download_instructions = """
NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
"""
```

File: easybuild/easyconfigs/n/NBO/NBO-7.0.10-gfbf-2022b.eb [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+homepage = 'http://nbo.chem.wisc.edu/'
+description = """ The Natural Bond Orbital (NBO) program is a discovery tool
+for chemical insights from complex wavefunctions. """
+
+toolchain = {'name': 'gfbf', 'version': '2022b'}
+toolchainopts = {'pic': True, 'i8': True}  # remove 'i8' if Gaussian was compiled with 32-bit integers
+
+# NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
+# The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/NBO/NBO-7.0.10-gfbf-2022b.eb:16]:
```suggestion
download_instructions = """
NBO7 is proprietary software, order here: https://charge.wisc.edu/chemistry/order_nbo7.aspx
The dowloaded tarball does not contain the bug-fix version so rename to nbo7.0.10-src.tar.gz
"""
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
I would also add `download_instructions` the easyconfigs.

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
hp - Linux Debian GNU/Linux 11, x86_64, 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz, Python 3.9.2
See https://gist.github.com/sassy-crick/c7cdbf698181f5021a4b8d7d6980e4c4 for a full test report.

**Comment by sassy-crick** [comment]
Done. I have added the install instructions as suggested, and uploaded the test report. I hope that helps

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**